test/spec/filters/messageForEventSpec.js

```diff
@@ -0,0 +1,58 @@
+'use strict';
+
+describe('messageForEventFilter', function(){
```

Comment:
Note that we could potentially make this even more generic with a `simplePrepositionFollowingVerb` filter, but I don't know that we need to go that far.